internal/proto/controller/api/resources/roles/v1/role.proto

```diff
@@ -25,7 +25,11 @@ message Principal {
 
 message GrantJson {
   // Output only. The ID, if set.
-  string id = 1; // @gotags: `class:"public"`
+  // Deprecated: use "ids" instead.
+  string id = 1 [deprecated = true]; // @gotags: `class:"public"`
+
+  // Output only. The IDs, if set.
+  repeated string ids = 4; // @gotags: `class:"public"`
```

Comment:
I had assumed we listed fields in protos in order of their field number but is that not the case?

Reply:
Generally we like to do that, which is why I also like to have buffer between numbers, e.g. 10, 20. But there's no buffer here and I thought it'd be more useful to have the field next to the deprecated one it's replacing.